README.md

```diff
@@ -55,6 +55,8 @@ To run locally:
 4. Run `scripts/write-config.sh`
 5. Run `./gradlew bootRun` to spin up the server.
 6. Navigate to [http://localhost:8080/#](http://localhost:8080/#)
+7. If this is your first time deploying to any environment, be sure to use the admin endpoint `/api/admin/v1/updatePipelineWorkspaceId/{pipelineName}/{workspaceId}` to set your pipeline's workspace id. Workspace id can be found through the terra ui workspace dashboard or through the Rawls GET workspace endpoint.
```

Comment:
workspace id actually doesn't show up (in an easily accessible way) on the terra ui workspace dashboard :(

Reply:
oh i usually just use the "bucket" name copy thing under "cloud information" to find the workspace id.  its usually like `sc-{workspace_uuid}` or `fc-{workspace_uuid}`.  I guess it depends on your definition of easily accessible hahaha

Reply:
fair - I get annoyed when I have to copy something into another text editor and edit it further. but you're right that it shows up there